README.md

```diff
@@ -131,7 +129,7 @@ commands will be ready to build:
 ```
 composer install --no-scripts
 composer vendor-expose
-npm install
+npm install --dev
```

Comment:
`--dev` is deprecated - if you're really wanting to install only dev packages, you want `--only=dev`.

Also

```suggestion
npm ci
```